wgpu-hal/src/vulkan/instance.rs

```diff
@@ -750,6 +750,17 @@ impl crate::Instance<super::Api> for super::Instance {
                 create_info = create_info.push_next(vk_create_info);
             }
 
+            let gpu_assisted_validation = &[vk::ValidationFeatureEnableEXT::GPU_ASSISTED];
+            let mut gpu_assisted_validation = vk::ValidationFeaturesEXT::builder()
+                .enabled_validation_features(gpu_assisted_validation)
+                .build();
+            if desc
+                .flags
+                .intersects(wgt::InstanceFlags::GPU_BASED_VALIDATION)
```

Comment:
We should also consider enabling syncval

Reply:
Uh...I actually don't know what that is! 👀💦

Reply:
VK_VALIDATION_FEATURE_ENABLE_SYNCHRONIZATION_VALIDATION_EXT - we should probably use VK_VALIDATION_FEATURE_ENABLE_GPU_ASSISTED_EXT, VK_VALIDATION_FEATURE_ENABLE_GPU_ASSISTED_RESERVE_BINDING_SLOT_EXT, and VK_VALIDATION_FEATURE_ENABLE_SYNCHRONIZATION_VALIDATION_EXT

Reply:
I _think_ I got this handled with bdfa467f7?